chainer/links/model/vision/vgg.py

```diff
@@ -66,26 +66,33 @@ class VGG16Layers(link.Chain):
             are not initialized by the pre-trained model, but the default
             initializer used in the original paper, i.e.,
             ``chainer.initializers.Normal(scale=0.01)``.
+        n_layers (int): The number of layers of this model. It should be
+            either 16 or 19.
 
     Attributes:
         available_layers (list of str): The list of available layer names
             used by ``forward`` and ``extract`` methods.
 
     """
 
-    def __init__(self, pretrained_model='auto'):
+    def __init__(self, pretrained_model, n_layers):
```

Comment:
In `__init__`,
* Why should the default value of argument `pretrained_model='auto'` be removed?
* The change/removal of comment seems unnecessary.

Reply:
So, This was because I did not want to define the default value for `n_layers` and now I know I should change the order of arguments.

Reply:
Ah, I missed the ResNet imple. I will follow the style.